src/wrapper/java_vm/vm.rs

```diff
@@ -36,6 +36,12 @@ use InitArgs;
 /// an appropriately-sized local frame for larger code fragments
 /// (see [`with_local_frame`](struct.JNIEnv.html#method.with_local_frame))
 /// and [auto locals](struct.JNIEnv.html#method.auto_local) in loops.
+///
+/// Jni-rs provides a helper struct to deal with such local references -
```

Comment:
I struggle to put in a clear and concise way :-) Basically, it provides JNIEnv to any native thread, combining the performance benefits of permanent attaches whilst removing the risk of local references leaks if used consistently. Is that correct that it is its main value proposition?